site/docs/providers/azure.md

```diff
@@ -64,23 +64,64 @@ providers:
 
 - `azure:chat:<deployment name>` - uses the given deployment (for chat endpoints such as gpt-35-turbo, gpt-4)
 - `azure:completion:<deployment name>` - uses the given deployment (for completion endpoints such as gpt-35-instruct)
+- `azure:moderation` - uses the Azure Content Safety API for content moderation
+
+## Azure Content Safety for Moderation
+
+The `azure:moderation` provider integrates with the Azure Content Safety API to provide content moderation capabilities. It analyzes text for harmful content and returns flags for any detected issues.
+
+### Environment Variables
+
+- `AZURE_CONTENT_SAFETY_ENDPOINT`: The endpoint URL for the Azure Content Safety service
+- `AZURE_CONTENT_SAFETY_API_VERSION`: The API version to use (default: "2024-09-01")
+- `AZURE_CONTENT_SAFETY_API_KEY`: The API key for Azure Content Safety service (preferred)
+- `AZURE_API_KEY`: The API key for Azure services (fallback)
+
+### Configuration
+
+```yaml
+providers:
```

Comment:
This is going to confuse people we should only configure it as a moderation provider and not put it in a provider block